src/test/user.spec.js

```diff
@@ -192,3 +200,139 @@ describe('Test user login, signup and account verification', () => {
     });
   });
 });
+
+describe('USER CAN FOLLOW OTHER USERS', () => {
+  beforeEach(async () => {
+    await db.User.destroy({ cascade: true, truncate: true });
+  });
+
+  it('should be able to follow other users', async () => {
+    const firstUser = await createUser(await getUser());
+    const secondUser = await createUser(await getUser());
+    await createUser(await getUser());
+    const token = jwt.sign(firstUser, process.env.JWT_SECRET);
+
+    const res = await server()
+      .patch(`${users}/follow`)
+      .set('token', token)
+      .send({ followeeId: secondUser.id });
+    expect(res.status).to.equal(200);
+  });
+
+  it('should not be able to follow a user twice at the same time', async () => {
+    const firstUser = await createUser(await getUser());
+    const secondUser = await createUser(await getUser());
+    await baseRepository.create(db.Follower, {
```

Comment:
use the public API instead.

Send the request `twice`. Assert that the user was followed once initially but the second request failed.